sklearn/linear_model/tests/test_least_angle.py

```diff
@@ -110,6 +141,8 @@ def test_all_precomputed():
             assert_array_almost_equal(expected, got)
 
 
+# FIXME: 'normalize' to be removed in 1.4
+@pytest.mark.filterwarnings("ignore:The default of 'normalize'")
```

Comment:
I personally would prefer to set `pytestmark = pytest.mark.filterwarnings(...)` at the beginning of the test module instead of setting the decorator on every test function.